python/tk_multi_publish2/dialog.py

```diff
@@ -785,13 +808,18 @@ def _synchronize_tree(self):
         the low level plugin representation
         """
         if len(self._plugin_manager.top_level_items) == 0:
-            # nothing in list. show the full screen drag and drop ui
-            self.ui.main_stack.setCurrentIndex(self.DRAG_SCREEN)
+            if not self.manual_load_enabled:
+                # No items collected and 'enable_manual_load' application option
+                # false, display that special error overlay.
+                self._show_no_items_error()
```

Comment:
👍

Reply:
The whole solution was simply to call `self.ui.main_stack.setCurrentIndex(self.PUBLISH_SCREEN)`